Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM node:latest
+
+MAINTAINER fokko@driesprong.frl
```

Comment:
Sure, the maintainer field can be arbitrary. It will fill in the author field at the Docker registry. Currently I have the whole-set up running using Docker as in https://github.com/nightscout/nightscout-docker